android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -495,6 +497,14 @@ fun configure(
         builder.observerMode(observerMode)
     }
 
+    if (verificationMode != null) {
+        try {
+            builder.entitlementVerificationMode(EntitlementVerificationMode.valueOf(verificationMode))
```

Comment:
@NachoSoto FYI, I updated the method here in your branch to the latest in the native SDK